docs/uplink/installation.md

```diff
@@ -0,0 +1,177 @@
+# Install inlets-uplink
+
+Inlets Uplink is a self-hosted solution for managing many tunnels for a team or company.
+
+Learn why we created Inlets Uplink and how it might be a good fit for you in the announcement blog post:
+
+- [Inlets Uplink for SaaS & Service Providers](https://inlets.dev/blog/2022/11/16/service-provider-uplinks.html)
+
+## Pre-reqs
+* A Kubernetes cluster with LoadBalancer capabilities (i.e. public cloud).
+* A domain name clients can use to connect to the tunnel control plane.
+* [arkade](https://github.com/alexellis/arkade), a simple CLI tool that provides a quick way to install various apps and download common binaries much quicker
+    
+    To install arkade run:
+
+    ```bash
+    curl -sSLf https://get.arkade.dev/ | sudo sh
+    ```
+
+## Create a Kubernetes cluster
+
+Create a cluster with at least 3 nodes 2-4GB of RAM each
+
```

Comment:
Let's make this Option A or B

There are two options for connecting clients .... etc ....

## A) Connect clients through Ingress

## B)